### ISSUE TITLE
ListView with same sorting header like in DataGrid helpwanted

### ISSUE THREAD

**Issue by tlostak** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#913]
Hello,
I would like to ask you, if is possible to have **ListView with sorting** items by clicking on their header. 
I mean, I want to implemenent same **sorting from DataGrid** where I just click on header and I got all items sorted (It looks very good and works perfect).

(Simply said - I need _CanUserSortColumns="True" in ListView)

I have these codes:
**ListView**
```
        <ListView x:Name="ListViewMain" Height="175" VerticalAlignment="Top" ScrollViewer.HorizontalScrollBarVisibility="Disabled" Margin="-527,487,1299,0">
            <ListView.View>
                <GridView x:Name="GridViewMain">
                    <GridViewColumn Header="ID" Width="Auto" DisplayMemberBinding="{Binding ID}"/>
                    <GridViewColumn Header="Přezdívka" Width="200" DisplayMemberBinding="{Binding Name}"/>
                    <GridViewColumn Header="Umístění" Width="Auto" DisplayMemberBinding="{Binding Placement}"/>
                    <GridViewColumn Header="Zabití (TOP 10)" Width="Auto" DisplayMemberBinding="{Binding TotalKills}"/>
                    <GridViewColumn Header="Celkové body" Width="200" DisplayMemberBinding="{Binding Points}"/>
                </GridView>
            </ListView.View>
        </ListView>
```
**DataGrid**
```
        <DataGrid x:Name="DataGridMain" SelectionMode="Single" SelectionUnit="FullRow" IsHitTestVisible="True" AutoGenerateColumns="False" HorizontalAlignment="Left" Height="792" VerticalAlignment="Top" Width="792" CanUserSortColumns="True" HorizontalScrollBarVisibility="Disabled">
            <DataGrid.Columns>
                <DataGridTextColumn Header="ID" Width="Auto" Binding="{Binding ID}"/>
                <DataGridTextColumn Header="Přezdívka" Width="100" Binding="{Binding Name}"/>
                <DataGridTextColumn Header="Umístění" Width="Auto" Binding="{Binding Placement}"/>
                <DataGridTextColumn Header="Zabití (TOP 10)" Width="Auto" Binding="{Binding TotalKills}"/>
                <DataGridTextColumn Header="Celkové body" Width="Auto" Binding="{Binding Points}"/>
            </DataGrid.Columns>
        </DataGrid>
```

Thank you

**Comment by Keboo** [comment]
You would have to manually implement the sorting on the ListView to get that behavior. However I think there is a simpler alternative. Why not simply make the DataGrid readonly? You can do this [by setting IsReadOnly to true](https://docs.microsoft.com/en-us/dotnet/api/system.windows.controls.datagrid?view=netframework-4.7.1#editing) on the DataGrid.

**Comment by HaLuZaR** [comment]
Thank you for your reply,
I tried this solution with isReadOnly but I have problem with selecting. I would like to have same select like in ListView. That means I just click only somewhere on row and I got all row selected. But on DataGrid I have all these effects while click / mouseover over the cells, and that doesnt look good when Its ReadOnly.

If It is not possible, I'm okay with that because for now I implemented few RadioButtons for select condition of sorting the table.
